src/tactic/finish.lean

```diff
@@ -409,29 +446,120 @@ open lean lean.parser interactive interactive.types
 local postfix `?`:9001 := optional
 local postfix *:9001 := many
 
-meta def clarify (hs : parse simp_arg_list) (cfg : auto_config := {}) : tactic unit :=
-do s ← mk_simp_set ff [] hs,
-   auto.clarify s cfg
+/--
+  `clarify[h1,...,hn] using [e1,...,en]` negates the goal, normalizes hypotheses
```

Comment:
Here any everywhere else, we usually put a space after the tactic, as in `simp [foo]` rather than `simp[foo]`. I think we should enforce that as uniform style. @jesse-michael-han would you mind changing those?